### ISSUE TITLE
[Question] HW6 GG server outputs error when submission is done.

### ISSUE THREAD

**Issue by loneo** [kaist-cp/cs431#594]
I have got full scores on the local server with grading script and is trying to submit the file on the GG server. I have followed the instructions and made zip file with the submit.sh file. However when I submit the file on the GG server, it shows error that I didn't see on my workspace and scores 0. I tried deleting the submission zip file and making them again many times, but it keeps showing the same error. I think it is from the past version of the code, but I am not sure. Could you please check? I have attached the error screenshot.
<img width="560" alt="error" src="https://user-images.githubusercontent.com/48512272/146418609-3f44e635-b354-4d15-b28c-744efc7ee1b4.png">

**Comment by tomtomjhj** [comment]
I think you are using the old template code.

**Comment by loneo** [comment]
I thought that I was using the old template code, but its weird because when I checked the last error on the image, I don't have 'HazardBag' on my mod.rs file, 82 line. and I think the old template is the code where the Hazard struct uses HazardBag, but mine uses Hash function. So I was guessing either my submission script is outdated, or maybe grading server works by attaching the + and - of the submitted codes, and it did not take care of updated mod.rs file.
If my template is outdated, how can I update it to the newest version? I already tried cargo update, but is there anything I need to do more?
